plugin/trino-hive/src/test/java/io/trino/plugin/hive/avro/TestAvroSchemaEvolutionWithPartitioning.java

```diff
@@ -0,0 +1,429 @@
+/*
+ * Copyright Starburst Data, Inc. All rights reserved.
+ *
+ * THIS IS UNPUBLISHED PROPRIETARY SOURCE CODE OF STARBURST DATA.
+ * The copyright notice above does not evidence any
+ * actual or intended publication of such source code.
+ *
+ * Redistribution of this material is strictly prohibited.
+ */
+package io.trino.plugin.hive.avro;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.plugin.hive.containers.HiveMinioDataLake;
+import io.trino.plugin.hive.s3.S3HiveQueryRunner;
+import io.trino.testing.AbstractTestQueryFramework;
+import io.trino.testing.QueryRunner;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.trino.testing.MaterializedResult.resultBuilder;
+import static java.lang.String.format;
+
+@Test
+public class TestAvroSchemaEvolutionWithPartitioning
```

Comment:
Do we need to rely on Minio or how about file based metastore and the data store in local FS ?

Reply:
Can we add additional tests in PT ?

Reply:
This test is baed on io.trino.tests.product.hive.TestAvroSchemaEvolution which is based on accessing files directly on FS. This case requires creating same tmp location during Test for Test host and internally in Hive docker to apply those which makes test more complex and error prone. 
It's easier when files are accessing by S3. No need to replicate location in both places. 
In terms of File Metastore it disallows creating tables with location attribute.

Reply:
As for the PT tests will add similar one